src/component/form/CaseElement.tsx

```diff
@@ -89,19 +105,65 @@ export const getExitName = (
     // Don't reassign func params
     let newExitName = exitName;
 
-    // Some operators don't expect args
-    if (newArgList.length >= 0 && !operatorConfig.categoryName) {
+    if (newArgList.length > 0 && !operatorConfig.categoryName) {
         newExitName = composeExitName(operatorConfig.type, newArgList);
-    } else {
+    } else if (operatorConfig.categoryName) {
```

Comment:
we want to pass the `exitName` param through if it doesn't meet these conditions